website/src/components/FeaturedBlogPosts.tsx

```diff
@@ -15,26 +14,24 @@ interface Props {
 export const FeaturedBlogPosts: React.FunctionComponent<Props> = ({ posts }) => (
     <div className="container">
         <div className="row text-dark justify-content-center">
-            {posts
-                .filter((post, i) => post.active)
-                .map(({ title, description, thumbnail, url, active }, i) => (
-                    <div className="blog-posts-home__post col-md-4 px-2">
-                        <div className="card">
-                            <div className="text-center">
-                                <a href={url}>
-                                    <img className="blog-posts-home__thumbnail py-3 px-3" src={thumbnail} />
-                                </a>
-                            </div>
-                            <div className="card-body">
-                                <h5 className="card-title">{title}</h5>
-                                <p className="card-text">{description}</p>
-                                <a href={url} className="btn btn-primary">
-                                    Read more
-                                </a>
-                            </div>
+            {posts.map(({ title, description, thumbnail, url, active }, i) => (
```

Comment:
Removed the filtering of inactive posts. Was a well-intentioned idea but needlessly complicates things.